include/party.h

```diff
@@ -5,21 +5,21 @@
 
 u32 SaveArray_Party_sizeof(void);
 u32 PartyCore_sizeof(void);
-PARTY * SaveArray_Party_Alloc(HeapID heapId);
-void SaveArray_Party_Init(PARTY *party);
-void InitPartyWithMaxSize(PARTY *party, int maxSize);
-BOOL AddMonToParty(PARTY *party, const Pokemon *mon);
-BOOL RemoveMonFromParty(PARTY *party, int slot);
-int GetPartyMaxCount(const PARTY *party);
-int GetPartyCount(const PARTY *party);
-Pokemon *GetPartyMonByIndex(PARTY *party, int slot);
-void Party_GetUnkSubSlot(const PARTY *party, PARTY_EXTRA_SUB *dest, int slot);
-void Party_SetUnkSubSlot(PARTY *party, const PARTY_EXTRA_SUB *src, int slot);
-void Party_ResetUnkSubSlot(PARTY *party, int slot);
-void Party_SafeCopyMonToSlot_ResetUnkSub(PARTY *party, int slot, Pokemon *src);
-BOOL Party_SwapSlots(PARTY *party, int slotA, int slotB);
-void Party_Copy(const PARTY *src, PARTY *dest);
-BOOL PartyHasMon(PARTY *party, u16 species);
-PARTY *SaveArray_PlayerParty_Get(SaveData *saveData);
+Party * SaveArray_Party_Alloc(HeapID heapId);
+void SaveArray_Party_Init(Party *party);
+void InitPartyWithMaxSize(Party *party, int maxSize);
+BOOL AddMonToParty(Party *party, const Pokemon *mon);
+BOOL RemoveMonFromParty(Party *party, int slot);
+int GetPartyMaxCount(const Party *party);
```

Comment:
Party_GetMaxCount etc for the rest of file

Reply:
done the above three but the rest needs to be done in another PR really